### PR TITLE
Adding Central Hub and Event Frontend components

### PR DESCRIPTION
**Pull request by TariqC7**
The Homepage component works fine and needs to be integrated - will be revisited soon. As for the Event/AddEvent component there is a network request error that I am getting when trying to add an event but the front end is there and works otherwise - also will be revisited soon. A profile page frontend screen was also added and can be found with the Central Hub component.

### CODE REVIEW

File: sia-react-native/screens/Home Page Frontend/screens/Profile2.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
```
Comment (ShumbaBrown) [inline sia-react-native/screens/Home Page Frontend/screens/Profile2.js]: A Profile page already exists.

File: sia-react-native/screens/Home Page Frontend/Assets/r.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+r
```
Comment (ShumbaBrown) [inline sia-react-native/screens/Home Page Frontend/Assets/r.txt]: What is the purpose of this file/

File: sia-react-native/screens/Home Page Frontend/Assets/icons/r.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+r
```
Comment (ShumbaBrown) [inline sia-react-native/screens/Home Page Frontend/Assets/icons/r.txt]: What is the purpose of this file?

File: sia-react-native/screens/Events/api.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
+import moment from 'moment';
+import Constants from 'expo-constants';
+import uuid from 'uuid'
+
+const { manifest } = Constants;
+const api = manifest.packagerOpts.dev
+    ? manifest.debuggerHost.split(`:`).shift().concat(`:19010`)
+    : `productionAPI.example.com`;
```
Comment (ShumbaBrown) [inline sia-react-native/screens/Events/api.js]: What is this API you are using? 
We have our own Backend component that acts as an events api. Use that.

File: sia-react-native/screens/Events/EventForm.js [hunk 5]
```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
```
Comment (ShumbaBrown) [inline sia-react-native/screens/Events/EventForm.js]: Why so many files? is it possible to consolidate?

File: sia-react-native/screens/Events/Event.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```
Comment (ShumbaBrown) [inline sia-react-native/screens/Events/Event.js]: What is the purpose of this file?

File: sia-react-native/screens/Home Page Frontend/screens/components/Hub/Home2.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
+import React, { Component } from "react";
```
Comment (ShumbaBrown) [inline sia-react-native/screens/Home Page Frontend/screens/components/Hub/Home2.js]: There is already a Home screen. Please merge

File: sia-react-native/screens/Events/EventCard.js [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+                        style={styles.counter}
+                    >
+                        <Text style={styles.counterText}>{seconds}</Text>
+                        <Text style={styles.counterLabel}>SECONDS</Text>
+                    </View>
+                </View>
+            </View>
+        </TouchableHighlight>
+    );
+}
```
Comment (ShumbaBrown) [inline sia-react-native/screens/Events/EventCard.js]: Events don't have these attributes. Why are you defining your own event? Use the definition Matthew made it is in the classes folder

**Comment by TariqC7** [comment]
Changes to be made

**Pull request closed without merging**